song/routers/postRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+
+const postController = require("../controllers/postController");
+
+router.get("/postinglist", postController.postingList);
+router.get("/userposting/:userId", postController.userPosting);
+
+router.post("/createpost", postController.createPost);
+router.patch("/modify/:userId", postController.modify);
+router.delete("/remove/:postId", postController.remove);
```

Comment:
```suggestion
router.post("", postController.createPost);
router.patch("/:userId", postController.modify);
router.delete("/:postId", postController.remove);
```
엔드포인트에는 동사(액션)이 들어가지 않습니다. what을 의미하는 목적자원만 적어주시면 됩니다. 지금 작성된 엔드포인트에 따르면 `/posts`라는 엔드포인트에 POST, GET, PATCH, DELETE 라는 http request method에 따라서 동작을 의미하는 how를 나타낼 수 있기 때문입니다. 자세한 내용은 금주에 예정되어 있는 REST 세션에서 다룰 예정이니 지금은 엔드포인트에 동사를 작성하는 것을 지양해야 한다는 것만 알아주세요~